acceptance-tests/tests/workflows/accountManagementFlow.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { describe, beforeAll, it, expect, afterAll } from 'vitest';
+import { TestState } from '../../lib/TestState';
+import { getInitPromptSequence, ENTER } from '../../lib/prompt';
+
+describe('Account Management Flow', () => {
+  let testState: TestState;
+  const accountName = 'test-account';
+
+  beforeAll(async () => {
+    testState = new TestState();
+  });
+
+  afterAll(() => {
+    testState.cleanup();
+  });
+
+  describe('initial state', () => {
+    it('config file should not exist', async () => {
+      expect(
+        testState.existsInTestOutputDirectory(
+          testState.getTestConfigFileNameRelativeToOutputDir()
+        )
+      ).toBe(false);
+    });
+  });
+
+  describe('hs init', () => {
+    it('should create a new config file', async () => {});
```

Comment:
Do we need this?

Reply:
Good catch, removed in [aa8facf](https://github.com/HubSpot/hubspot-cli/pull/1200/commits/aa8facfdf6ade43b9dc39b3533031b715dadecad)